server/controllers/reviews.js

```diff
@@ -0,0 +1,128 @@
+const asyncHandler = require("express-async-handler");
+const Reviews = require("./../models/Review");
+const Request = require("./../models/Request");
+const Profile = require("./../models/Profile");
+const mongoose = require("mongoose");
+
+exports.getReviews = asyncHandler(async (req, res) => {
+  const { profileId } = req.params;
+
+  if (!mongoose.isValidObjectId(profileId)) {
+    res.status(400);
+    throw new Error("Invalid profile ID");
+  }
+
+  const profile = await Profile.findById(profileId);
+
+  if (!profile) {
+    res.status(404);
+    throw new Error("No profile found for this ID");
+  }
+
+  const reviews = await Reviews.find({ reviewee: profileId })
+    .sort({ createdAt: -1 })
+    .populate("reviewer", {
+      firstName: 1,
+      lastName: 1,
+      profilePic: 1,
+    });
+  res.status(200).send(reviews);
+});
+
+exports.createUpdateReviews = asyncHandler(async (req, res) => {
+  const { id: reviewerUserId } = req.user;
+  const { requestId } = req.params;
+  const { reviewee, comment, starRating } = req.body;
```

Comment:
Validate these fields and send a `400` if any is missing. A good practice :smiley:

Reply:
I implemented the validation of those fields in a separate middleware function, [you can check that out here](https://github.com/hatchways/team-brioche/pull/87/files#diff-6e9ae97ce4941e1249132aea512a5590992e2143a90eaf9efaf285e5501970f1R10).